wp/lib/bap_wp/src/bil_to_bir.mli

```diff
@@ -49,4 +49,8 @@ val mk_assert_fail : unit -> Bap.Std.Sub.t * Bap.Std.Exp.t
 
 (** Take a BIL program (a list of statements) and turn it into a
     subroutine. *)
-val bil_to_sub : Bap.Std.Bil.t -> Bap.Std.Sub.t
\ No newline at end of file
+val bil_to_sub : Bap.Std.Bil.t -> Bap.Std.Sub.t
+
+(** Take a sub term and a type for the term to build a BIL
```

Comment:
I'd say [sub_t term] (with the brackets so it's pretty in the docs) or just "a subroutine".

Reply:
More generally, I'd

A) `open Bap.Std` so it doesn't need to be prefixed everywhere

B) Use the "nice names" everywhere, so `sub term` instead of `Sub.t` etc.

I understand this is a bit of scope creep, since it doesn't directly pertain to the added functionality. Maybe @jtpaasch will tell me that's Not Ok.

Reply:
> A) open Bap.Std so it doesn't need to be prefixed everywhere
>
> B) Use the "nice names" everywhere, so sub term instead of Sub.t etc.

It was looking pretty cluttered with the `Bap.Std`prefix and `t`s.

Reply:
@codyroux `B)` might cause style mismatching with `bil_to_bir.ml`. For example, I could change 
`mk_assert_fail : unit -> Sub.t * Exp.t` to `mk_assert_fail : unit -> sub term * Exp.t`, though in `bil_to_bir.ml` we have it written as `let mk_assert_fail () : Sub.t * Exp.t =`.

Reply:
It doesn't matter, the `.ml` is for our eyes only, but the `mli` is for any client of the module. Also, it would be `Bil.exp` rather than `Exp.t`.

But you can change the `.ml` also, if you like.